inc/saitam.h

```diff
@@ -1076,6 +1121,34 @@ typedef enum _sai_tam_tel_type_attr_t
      */
     SAI_TAM_TEL_TYPE_ATTR_COUNTER_SUBSCRIPTION_LIST,
 
+    /**
+     * @brief The mode of TAM telemetry type
+     *
+     * @type sai_tam_tel_type_mode_t
+     * @flags CREATE_ONLY
+     * @default SAI_TAM_TEL_TYPE_MODE_SINGLE_TYPE
+     */
```

Comment:
For clarity, it should be marked as:
@validonly SAI_TAM_TEL_TYPE_ATTR_TAM_TELEMETRY_TYPE == SAI_TAM_TELEMETRY_TYPE_COUNTER_SUBSCRIPTION

Reply:
good idea，thanks for your suggestion.